packages/react-google-analytics/src/tests/Universal.test.tsx

```diff
@@ -138,12 +139,59 @@ describe('<Universal />', () => {
   });
 
   describe('debug', () => {
-    it('sets the debug mode', () => {
+    it('not setting debug should load script without restrictions', () => {
```

Comment:
This should read as a sentence.
`it loads scripts without restrictions when debug is not set`